.gitignore

```diff
@@ -1,9 +1,14 @@
 *.pyc
 __pycache__/
+data/
+venv/
 dist/
 .DS_Store
 *.egg-info
 build
 run-info.txt
 plays
 *.so
+.*.swp
+wandb/
+./tmp/
```

Comment:
Are we creating `./tmp/` directories anywhere at this point? If it's just on your machine, I'd prefer to not add it to this file and instead just add a `.gitignore => *` file in the directory you want to avoid.

Reply:
We don't formally make that directory, but we incentivise it -- it's a natural consequence of the minimalist comand: `start-training tmp` :)

Reply:
Arguably if we didn't want that behaviour we should be defaulting to putting training runs under `data/training-run-name`?

Reply:
FYI, you can also run `./start-training data/tmp` and get the same effect. Good point though, happy to have it in gitignore.